src/Main.java

```diff
@@ -1,5 +1,64 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        int [] arr = new int[]{1, 2, 3};
```

Comment:
здесь нужно было создать массив длины 3 и потом заполнить его элементы, потому что как сейчас - это то же самое, что
```suggestion
        int [] arr = {1, 2, 3};
```